doc/UserGuide/BuildingRunningTesting/TestingUPP.rst

```diff
@@ -14,32 +14,47 @@ Running UPP Regression Tests
 
 To run the full regression test (RT) suite in preparation for opening a pull request (PR):
 
-   #. Navigate to the local clone of your UPP fork containing the changes you would like to introduce, and run the included RT script within ``/ci``
+   #. Navigate to the local clone of your UPP fork containing the changes you would like to introduce, and run the included RT script within ``ci``:
 
-     .. code-block:: console
+      .. code-block:: console
 
-        cd /path/to/UPP/ci
-        nohup ./rt.sh -a <my_account> -r $PWD/rundir -t $PWD/../ &
+         cd /path/to/UPP/ci
+         nohup ./rt.sh -a <my_account> -r $PWD/rundir -t $PWD/../ &
```

Comment:
@jdkublnick Below, we tell them they can ignore the `-t` option, but I've always run with it. Let me check whether we can just omit it here. If we need it, the "Additional Configuration" section will need a change.